examples/getting-started-session-based/03-serving-session-based-model-with-Torch-backend.ipynb

```diff
@@ -0,0 +1,1473 @@
+{
```

Comment:
<p>I am confused - what is the flow of the notebooks?</p><p> </p><p>Will this notebook continue from the 02-session-based notebook? In this notebook, we will train another model from scratch?</p><p>Should this notebook load the saved model? Or do we replace the 02 notebook with it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Transformers4Rec/pull/552/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Yes it is how we do now. we dont retrain we load the trained and saved model.